### ISSUE TITLE
We should consider constructiong multivisualization tables

### ISSUE THREAD

**Issue by sylwiabr** [luna/luna-studio#1164]
Please ensure that you check the latest version of Luna Studio to see if your feature has been implemented.

### General Summary
![obraz](https://user-images.githubusercontent.com/12892578/46415083-d02a8e00-c724-11e8-95c5-d0ace529e242.png)
I think we should make tables like kaggle or IBM watson analytics with optional sorting/filters/simple plots/histograms

### Motivation

- Better description of data in the table

**Comment by wdanilo** [comment]
Yes! Few things from my side. Such tables need to contain other visualizations like sliders. Consider the following IBM Watson screenshots as ideas reference:
![table1](https://user-images.githubusercontent.com/1623053/46415748-36fc7700-c726-11e8-8394-6e0b2acc4038.PNG)
![table2](https://user-images.githubusercontent.com/1623053/46415749-36fc7700-c726-11e8-98e7-b893060864a2.png)
![table3](https://user-images.githubusercontent.com/1623053/46415750-36fc7700-c726-11e8-8ae9-bf59cf8409eb.jpg)
